### PR TITLE
Add show tables with schema

### PR DESCRIPTION
**Pull request by apstndb**
This PR adds named schemas support as `SHOW TABLES <schema>` command.
This is implemented as an extension of the existing `SHOW TABLES` to `SHOW TABLES [<schema>]`, but it is explained as another command in README.md.

```
spanner> SHOW TABLES;
+----------------------------------------+
| Tables_in_apstndb-sampledb3            |
+----------------------------------------+
| Albums                                 |
| Concerts                               |
| Singers                                |
| Songs                                  |
+----------------------------------------+
71 rows in set (0.18 sec)

spanner> SHOW TABLES sch1;
+-----------------------------+
| Tables_in_apstndb-sampledb3 |
+-----------------------------+
| Albums                      |
| Singers                     |
| SingerView                  |
| Songs                       |
+-----------------------------+
4 rows in set (0.18 sec)
```

Related #172

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,7 +192,8 @@ and `{}` for a mutually exclusive keyword.
 | Switch database | `USE <database> [ROLE <role>];` | The role you set is used for accessing with [fine-grained access control](https://cloud.google.com/spanner/docs/fgac-about). |
 | Create database | `CREATE DATABSE <database>;` | |
 | Drop database | `DROP DATABASE <database>;` | |
-| List tables | `SHOW TABLES;` | |
+| List tables in a default schema| `SHOW TABLES;` | |
+| List tables in a named schema | `SHOW TABLES <schema>;` | |
```
Comment (yfuruyama) [inline README.md:196]: Maybe we can combine two rows into one like this.

```
| List tables in a schema | `SHOW TABLES [<schema>];` | If schema is not provided, default schema is used |
```

File: README.md [hunk 2]
```diff
@@ -218,8 +219,8 @@ and `{}` for a mutually exclusive keyword.
 | Commit Read-Write Transaction | `COMMIT;` | |
 | Rollback Read-Write Transaction | `ROLLBACK;` | |
 | Start Read-Only Transaction | `BEGIN RO [{<seconds>\|<RFC3339-formatted time>}] [PRIORITY {HIGH\|MEDIUM\|LOW}] [TAG <tag>];` | `<seconds>` and `<RFC3339-formatted time>` is used for stale read. See [Request Priority](#request-priority) for details on the priority. The tag you set is used as request tag. See also [Transaction Tags and Request Tags](#transaction-tags-and-request-tags).|
-| End Read-Only Transaction | `CLOSE;` | |
-| Exit CLI | `EXIT;` | |
+| End Read-Only Transaction | `CLOSE;`                                                                                       | |
```
Comment (yfuruyama) [inline README.md:222]: Is this changed by some formatter?

File: README.md [hunk 3]
```diff
@@ -218,8 +219,8 @@ and `{}` for a mutually exclusive keyword.
...
+| Exit CLI | `EXIT;`                                                                                        | |
```
Comment (yfuruyama) [inline README.md:223]: ditto

**Review by yfuruyama — APPROVED** [review]
Left some minor comments, but LGTM. Thanks for submitting this change!

File: README.md (hunk 2 above)
Comment (apstndb) [reply README.md]: Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/173/commits/0d7393de10bf1a597b0276803a03ec9da1761dca

File: README.md (hunk 1 above)
Comment (apstndb) [reply README.md]: Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/173/commits/0d7393de10bf1a597b0276803a03ec9da1761dca

**Comment by yfuruyama** [comment]
Thanks for the additional change. LGTM!

**Pull request merged**